controller/persistence/cluster_store.go

```diff
@@ -46,6 +46,7 @@ func (entity *Cluster) GetEntityType() string {
 type ClusterStore interface {
 	Store
```

Comment:
Is this still used?

Reply:
Only by migration code. I've been trying to reduce the entangling of the deprecated stores, but there's more I can do. Also, I figure we can strip everything out in Ziti 1.1.